packages/devtools_app/lib/src/shared/feature_flags.dart

```diff
@@ -99,6 +99,11 @@ abstract class FeatureFlags {
   /// https://github.com/flutter/devtools/issues/7856
   static bool wasmOptInSetting = true;
 
+  /// Flag to enable the Flutter Property Editor sidebar.
+  ///
+  /// https://github.com/flutter/devtools/issues/7854
+  static bool propertyEditor = enableExperiments;
```

Comment:
add this to feature_flags_test.dart

Reply:
Done! Also added a test for the `inspectorV2` flag